src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,65 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  mov: Movie;
```

Comment:
```suggestion
  movie: Movie;
```
do not make variable name shorter